crates/wallet/tests/wallet.rs

```diff
@@ -553,6 +553,17 @@ fn test_create_tx_default_locktime_cltv() {
     assert_eq!(psbt.unsigned_tx.lock_time.to_consensus_u32(), 100_000);
 }
 
+#[test]
+fn test_create_tx_locktime_cltv_for_specific_time() {
+    let (mut wallet, _) = get_funded_wallet_single(get_test_single_sig_cltv_for_specific_time());
+    let addr = wallet.next_unused_address(KeychainKind::External);
+    let mut builder = wallet.build_tx();
+    builder.add_recipient(addr.script_pubkey(), Amount::from_sat(25_000));
+    let psbt = builder.finish().unwrap();
```

Comment:
To check that finalization works, you'll want to sign the PSBT also.

```suggestion
    let mut psbt = builder.finish().unwrap();
    let finalized = wallet.sign(&mut psbt, SignOptions::default()).unwrap();
    assert!(finalized);
```